apps/website/app/utils/supabase/server.ts

```diff
@@ -0,0 +1,46 @@
+import { createServerClient, type CookieOptions } from "@supabase/ssr";
+import { cookies } from "next/headers";
+import { Database } from "~/utils/supabase/types.gen";
+
+export const createClient = async () => {
+  const cookieStore = await cookies();
+  const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL;
+  const supabaseKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY;
+
+  if (!supabaseUrl || !supabaseKey) {
+    throw new Error("Missing required Supabase environment variables");
+  }
+
+  return createServerClient<Database>(supabaseUrl, supabaseKey, {
+    cookies: {
```

Comment:
Is this planned to be used in the future for a specific feature / use case?

Reply:
No idea about that part of code. @sid597 ?